students/scotchwsplenda/Lesson_5/MailRoom_3.py

```diff
@@ -65,7 +64,7 @@ def send_note():
                     donators.update({donor_inp: [new_don]})
                     send_thanks(donor_inp, new_don)
             if respondy == 3:
-                mainy()
+                exit_program()
```

Comment:
your program is now exiting everytime you do one selection.

your function should just simply `return` when they are done performing whatever logic this function should perform.

Reply:
yeah that worked!